downloads/release-notes/1.2.7/RELEASE_NOTE.md

```diff
@@ -22,9 +22,92 @@ However, you need to use the following commands instead of the above if you have
 
 If you have not installed jBallerina, then download the [installers](https://ballerina.io/downloads/) to install.
 
+## Language
+
 ## Standard Library
-- 
 
-## Dev Tools
-- 
+### I/O
+
+When a CSV file has an empty record, the previous implementation throws a null pointer exception and it was fixed with this release.
```

Comment:
NPE is java level issue. Should we re-word it with something related to ballerina error?

Reply:
Done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1140#discussion_r469204240.